diff --git a/src/config/server.ts b/src/config/server.ts
index 0cce3185..e2f067b7 100644
--- a/src/config/server.ts
+++ b/src/config/server.ts
@@ -185,8 +185,8 @@ const SERVER_CONFIG: StrictServerConfiguration = {
     patchNetworkAccountSyncFixes: true,
     enableShardKeyChanges: true,
     verticalScalingMode: {
-      enabled: false,
-      path: '.'
+      enabled: true,
+      path: '/home/node/config'
     }
   },
   ip: {
